dcc-submission-server/src/main/java/org/icgc/dcc/submission/server/web/controller/NextReleaseController.java

```diff
@@ -333,4 +344,17 @@ private boolean isAccessible(Authentication authentication) {
     return systemService.isEnabled() || isSuperUser(authentication);
   }
 
+  private void checkProjectsWithTransfers(List<QueuedProject> queuedProjects) {
+    val transfers = queuedProjects.stream()
```

Comment:
I'm thinking this should be pushed into the service layer as we discussed. Reason being is that we should guard against this for other actions such as "signoff" or "release".